### DISCUSSION TITLE
Can I make TabView header scroll?

### DISCUSSION THREAD

**Discussion by andreas-spindler-mw** [enisn/UraniumUI#637] [General]
Hi there,
is there any way I can make the default header of the `TabView` scroll, if there are more tabs then space? Currenttly, all the tabs are squashed into the available width.

Thank you, for your support.

**Comment by andyzukunft** [comment]
I think https://github.com/enisn/UraniumUI/issues/693 will address your question?

**Reply by andreas-spindler-mw** [reply]
Yes! 
But unfortunately too late for us, as we already build a workaround with CollectionViews.

**Comment by enisn** [comment]
Implemented in https://github.com/enisn/UraniumUI/pull/725

**Comment by enisn** [comment]
Since it's released in v2.9, I'm closing this discussion
